Hazel/src/Hazel/Core/Input.cpp

```diff
@@ -11,7 +13,11 @@ namespace Hazel {
 
 	Scope<Input> Input::Create()
 	{
-	#ifdef HZ_PLATFORM_WINDOWS
+	#if defined(HZ_PLATFORM_WINDOWS)
+		return CreateScope<WindowsInput>();
+	#elif defined(HZ_PLATFORM_LINUX)
```

Comment:
These functions are delegated to a library that is cross-platform. Adding compiler directive branches is unneeded and only convolutes Hazel's source code.

Reply:
This is suggestion by @LovelySanta. Linux version is using Windows implementation since that is cross-platform, but really it should use its own implementation, like LinuxInput and etc.

Reply:
This is suggestion by @LovelySanta. Linux version is using Windows implementation since that is cross-platform, but really it should use its own implementation, like LinuxInput and etc.

Reply:
I think that would be something for Cherno to implement tho...